homeassistant/components/zha/sensor.py

```diff
@@ -273,7 +262,7 @@ class ElectricalMeasurement(Sensor):
     _attr_should_poll = True  # BaseZhaEntity defaults to False
     _attr_state_class: SensorStateClass = SensorStateClass.MEASUREMENT
     _attr_name: str = "Active power"
-    _unit = POWER_WATT
+    _attr_native_unit_of_measurement: str = UnitOfPower.WATT
```

Comment:
Need to specify the type here due to funny inheritance on `ElectricalMeasurement` class